<ui version="4.0" >
 <class>qt4subeditor</class>
 <widget class="QMainWindow" name="qt4subeditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Subtitle Editor</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="subeditor.qrc" >:/pix/pix/subedit_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Movie Control</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pbOpenMovie" >
           <property name="toolTip" >
            <string>Open movie</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/fileopen.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbPlayPauseMovie" >
           <property name="toolTip" >
            <string>Play movie</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/player_play.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbFastBack" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Back 10 mn</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/player_start.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbBack" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Back 1 mn</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/player_rew.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbForward" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Forward 1mn</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/player_fwd.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbFastForward" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Forward 10mn</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/player_end.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="lblLength" >
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="text" >
            <string>length</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbNewSubBegin" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Begining of new subtitle</string>
           </property>
           <property name="text" >
            <string>|&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbNewSubEnd" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>End of new subtitle</string>
           </property>
           <property name="text" >
            <string>>|</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbSubtitle" >
      <property name="title" >
       <string>Subtitle</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pbNewSubtitle" >
           <property name="toolTip" >
            <string>New subtile file</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/filenew.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbOpenSubtitle" >
           <property name="toolTip" >
            <string>Open subtitle file</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/fileopen.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSaveSubtitle" >
           <property name="toolTip" >
            <string>Save subtitle file</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/filesave.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSaveSubtitleAs" >
           <property name="toolTip" >
            <string>Save as</string>
           </property>
           <property name="text" >
            <string>S&amp;ave as</string>
           </property>
           <property name="icon" >
            <iconset resource="subeditor.qrc" >:/pix/pix/filesave.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbApply" >
           <property name="toolTip" >
            <string>Apply subtitle fiel to movie</string>
           </property>
           <property name="text" >
            <string>Apply to movie</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="lvSubs" />
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit" >
        <property name="toolTip" >
         <string>Exit</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="subeditor.qrc" >:/pix/pix/exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="subeditor.qrc" />
 </resources>
 <connections/>
</ui>
